README.md

```diff
@@ -40,6 +40,16 @@ ln -sf $(pwd)/streamr-docker-dev/bin.sh /usr/local/bin/streamr-docker-dev
 
 The docker stack has not been tested in a Windows environment and is not recommended at this time.
 
+If you know what services you need, you don't need to use the `bin.sh`, you can just use `docker-compose` directly, like so:
+
+1. Install and start Docker service.
+
+2. Clone this repo: `git clone git@github.com:streamr-dev/streamr-docker-dev.git`, change into that directory `cd streamr-docker-dev`
+
+3. Bind the loopback interface to 10.200.10.1: `netsh int ip add address "Loopback" 10.200.10.1`
```

Comment:
Should the `bin.sh` script be updated with this?

Reply:
after there is some idea how to run .sh scripts in windows. Needs more research, maybe it's WSL, maybe something else. Maybe there should be a .bat version of that script. I don't know.